inference/worker/chat_chain_utils.py

```diff
@@ -11,7 +11,7 @@
 from langchain.prompts import PromptTemplate
 from loguru import logger
 from oasst_shared.schemas import inference
-from opeanapi_parser import prepare_plugin_for_llm
+from openapi_parser import prepare_plugin_for_llm
```

Comment:
Interesting! Why wasn't this detected during use and importing of the module? Does the current design fail quietly?

Reply:
No, this wasn't broken, the module was also actually named `opeanapi_parser`, I just renamed it